geoips/plugins/modules/readers/seviri_hrit.py

```diff
@@ -28,13 +28,13 @@
 import os
 import logging
 import numpy as np
-from .utils.hrit_reader import HritFile, HritError
+from geoips.plugins.modules.readers.utils.hrit_reader import HritFile, HritError
 
 # Installed Libraries
 
 # GeoIPS Libraries
 from geoips.filenames.base_paths import PATHS as gpaths
-from geoips.plugins.modules.readers.utils.geostationary_geolocation import (
+from .utils.geostationary_geolocation import (
     get_geolocation,
 )
```

Comment:
Why do the changes on lines 31 and 37 appear to be opposite changes? Is there a preference here?

Reply:
I suppose relative imports is probably preferable to imports with the full path?  Though I'm not sure it matters too much?  I think at one stage of my bug fixing with the plugin registries, etc one or the other didn't work, but it appears now that both work if there is one of each in here...  I think ignore this for now, and resolve the conversation.

Reply:
Okay, I'll resolve since it works. Just odd to make the opposite change on lines in the same file.